pkg/service/repair/service.go

```diff
@@ -589,16 +585,6 @@ func (s *Service) hostRangeLimits(ctx context.Context, client *scyllaclient.Clie
 	return out, err
 }
 
-func (s *Service) maxRepairRangesInParallel(shards uint, totalMemory int64) int {
```

Comment:
revert and use above function, as per Asias this is OK
(it can be reverted to REST call later, so TODO can be moved here)